### PR TITLE
CMCL-0000: Input system improvements

### PR DESCRIPTION
**Pull request by AntoineCharton**
- Add a PlayerCount script to count players and place them.
- Fix joystick right stick scale on Input map
- Fix Car multiplayer scene
- Fix Player not jumping
- Other small improvements
- Remove untested action maps

### Testing status

- [ ] Passed all automated tests
- [ ] Manually tested

### CODE REVIEW

**Comment by glabute** [comment]
It's difficult to review these things when you commit a bunch of unrelated metafile changes.  Can you please revert the file changes that are not related to the thing being implemented.

File: com.unity.cinemachine/Samples~/Input System Samples/Split Screen Multiplayer/PlayerInitializer.cs [hunk 1]
```diff
@@ -9,7 +9,7 @@ public class PlayerInitializer: MonoBehaviour
         void Start()
         {
             // Initializing the start position of the player.
-            transform.position = new Vector3(CinemachineBrain.ActiveBrainCount * 2, 2, 0);
+            transform.position = new Vector3(PlayerCounter.PlayerCount * 2, 2, 0);
 
             // Shift one bit per brain Count.
             m_CinemachineBrain.ChannelMask = (OutputChannel.Channels)(1 << CinemachineBrain.ActiveBrainCount);
```
Comment (glabute) [inline com.unity.cinemachine/Samples~/Input System Samples/Split Screen Multiplayer/PlayerInitializer.cs:15]: why not use PlayerCounter here too?

File: com.unity.cinemachine/Samples~/Input System Samples/Split Screen Multiplayer/PlayerCounter.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
+using UnityEngine;
+
+public class PlayerCounter : MonoBehaviour
```
Comment (glabute) [inline com.unity.cinemachine/Samples~/Input System Samples/Split Screen Multiplayer/PlayerCounter.cs:3]: missing namespace

**Review by glabute — CHANGES_REQUESTED** [review]
Please revert irrelevant file mods

File: com.unity.cinemachine/Samples~/Input System Samples/Split Screen Multiplayer/PlayerInitializer.cs (hunk 1 above)
Comment (AntoineCharton) [reply com.unity.cinemachine/Samples~/Input System Samples/Split Screen Multiplayer/PlayerInitializer.cs:15]: Could make the script more reusable outside of the context of the sample. For example if there is a different number of brain. Do you want the change? I don't mind restraining the usage.

File: com.unity.cinemachine/Samples~/Input System Samples/Split Screen Multiplayer/PlayerCounter.cs (hunk 2 above)
Comment (AntoineCharton) [reply com.unity.cinemachine/Samples~/Input System Samples/Split Screen Multiplayer/PlayerCounter.cs]: Fixed

File: com.unity.cinemachine/Samples~/Input System Samples/Split Screen Multiplayer/PlayerInitializer.cs (hunk 1 above)
Comment (glabute) [reply com.unity.cinemachine/Samples~/Input System Samples/Split Screen Multiplayer/PlayerInitializer.cs:15]: I think it's confusing this way.  Let PlayerCounter be the single source for player count.

Comment (AntoineCharton) [reply com.unity.cinemachine/Samples~/Input System Samples/Split Screen Multiplayer/PlayerInitializer.cs]: Fixed

**Review by glabute — APPROVED** [review]

**Pull request merged**